pkgs/applications/misc/furtherance/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, appstream-glib, desktop-file-utils
+, glib, libadwaita, meson, ninja, pkg-config, python3, wrapGAppsHook4, dbus
+, gtk4, sqlite }:
+
+stdenv.mkDerivation rec {
+  pname = "furtherance";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "lakoliu";
+    repo = "Furtherance";
+    rev = "v${version}";
+    sha256 = "oIzwFPJUY+OwT2B6n7NPg8bMxEd4D0MKPe3GQxrXmEU=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    sha256 = "FEvRcUzV5gk9NU/pMOLFLsMfZq6nMsKLRvpFR14W/eU=";
+  };
+
+  nativeBuildInputs = [
+    appstream-glib
+    desktop-file-utils
+    meson
+    ninja
+    pkg-config
+    python3
+    rustPlatform.cargoSetupHook
+    rustPlatform.rust.cargo
+    rustPlatform.rust.rustc
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [ dbus glib gtk4 libadwaita sqlite ];
+
+  enableParallelBuilding = true;
+
+  preInstall = ''
+    chmod +x ../build-aux/meson/postinstall.py
+    patchShebangs ../build-aux/meson/postinstall.py
+    substituteInPlace ../build-aux/meson/postinstall.py \
+      --replace /usr/local $out
```

Comment:
This substitution is not necessary, it is only used as a [fallback](https://github.com/lakoliu/Furtherance/blob/996f669796e83f08fd92506441e0340c0f393f61/build-aux/meson/postinstall.py#L6) – Meson [will](https://mesonbuild.com/Reference-manual_builtin_meson.html#mesonadd_install_script) [always](https://github.com/mesonbuild/meson/blob/2e3ac3eec00f32dabcfc470cd4c1090303d08e58/mesonbuild/minstall.py#L646) set `MESON_INSTALL_PREFIX` correctly.